CODEOWNERS

```
@@ -1,9 +1,9 @@
 # https://help.github.com/en/articles/about-code-owners
 
-* @hmcts/platform-engineering @hmcts/devops @hmcts/ia-aip-frontend
+* @hmcts/platform-engineering @hmcts/devops @hmcts/immigration-asylum-admins
```

only rpe and devops should be default owners
```suggestion
* @hmcts/platform-engineering @hmcts/devops 
```